src/components/sections/secondary-banner.json

```diff
@@ -0,0 +1,72 @@
+{
+  "collectionName": "components_sections_secondary_banners",
+  "info": {
+    "displayName": "secondaryBanner",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
+    "name": {
+      "type": "string",
+      "unique": false,
+      "maxLength": 256,
+      "required": true
+    },
+    "title": {
```

Comment:
Suggestion: 
1. Let's modify `heading` component and add `level` property (number) which we will map on the FE side.
2. Let's use heading here.

Reply:
I see. But what about business requirement for this field? :
'limited to 256 symbols' 

I do not such restriction in `heading` component `text` field

Reply:
I think it should be clarified with Iryna. But even logically we don't need 256 symbols for h1...h5.

Reply:
In this way, do you mind setting 256 symbols restriction for `heading` component `text` field?

Reply:
updated!
Thanks